modules/navigation.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+export const navigation = () => {
+  const header = document.getElementById('menu');
+  const btns = header.getElementsByClassName('btn');
+
+  for (let i = 0; i < btns.length; i += 1) {
+    btns[i].addEventListener('click', function () {
```

Comment:
- [ ] Kindly use ES6 syntax (arrow) for your functions.

Reply:
@TedLivist Thank you for your review and suggestion. I have tried using the arrow function here several times, but it doesn't work. I tried to find a solution by searching different places, but according to w3schools, we should use the arrow function if the general function does not provide the expected output. In this case, if using the arrow function, the function does not work. Here is a reference link...

https://www.w3schools.com/js/js_arrow_function.asp